src/UI/Art/SpriteArtist/SpriteLibrary.java

```diff
@@ -288,17 +278,103 @@ public static Sprite getMoveCursorArrowSprite()
 
     return moveCursorArrowSprite;
   }
+  
+  /**
+   * Sets up the available palettes for use.
+   * Reads in the first 6 colors on each row.
+   * Bottom colors are buildings, top are units.
+   * The key color is the last color on the top row.
+   */
+  private static void initResources()
```

Comment:
`initResources` is too general - it's just handling the colors and factions.

Reply:
Since I'm not sure my phrasing is clear - it's just the name that is too general; maybe something like `loadFactionsAndColors` or whatever, but `initResources` is less descriptive than I prefer.